src/styles/blocks/header.scss

```diff
@@ -0,0 +1,157 @@
+.header {
+  background: url(../images/image.svg),
+    linear-gradient(107.56deg, #191536 0%, #000 100%);
+  background-size: contain, cover;
+  background-position:
+    0 44px,
+    center;
+  background-repeat: no-repeat, no-repeat;
+
+  @include on-tablet {
+    background: url(../images/image2.svg),
+      linear-gradient(107.56deg, #191536 0%, #000 100%);
+    background-size: unset, cover;
+    background-position:
+      top right,
+      center;
+    background-repeat: no-repeat, no-repeat;
+  }
+
+  @include on-desktop {
+    background: linear-gradient(107.56deg, #191536 0%, #000 100%);
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    box-sizing: border-box;
+    height: 100vh;
```

Comment:
this is causing weird UI on mobile, with a huge space between two section as following in the iamgeimage below:

<img width="450" alt="Screenshot 2024-12-05 at 09 38 22" src="https://github.com/user-attachments/assets/a0b7aa51-a7d6-4df7-89a6-3ecdb7fd469f">

Reply:
Resolvi utilizando height: auto ao invés de 100vh